CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
 #### Enhancements
 
+* Add optional filename verification to the `file_header` rule.  
```

Comment:
can you please add an explanation on `SWIFTLINT_CURRENT_FILENAME`? I think it'd be nice to also update the rule description to mention it.